src/locales/en/parsers/ENTimeUnitWithinFormatParser.ts

```diff
@@ -4,9 +4,11 @@ import { ParsingComponents } from "../../../results";
 import { AbstractParserWithWordBoundaryChecking } from "../../../common/parsers/AbstractParserWithWordBoundary";
 
 export default class ENTimeUnitWithinFormatParser extends AbstractParserWithWordBoundaryChecking {
-    innerPattern(): RegExp {
+    innerPattern(context: ParsingContext): RegExp {
+        const keywordOptional = context.option.forwardDate ? "?" : "";
         return new RegExp(
-            `(?:within|in|for)\\s*(?:(?:about|around|roughly|approximately|just)\\s*(?:~\\s*)?)?` +
+            `(?:within|in|for)${keywordOptional}` +
```

Comment:
I found appending the optional sign to the regex is more difficult to understand what will happens.

Could you please write the full question-marked prefix with you condition? 
```
const prefix= context.option.forwardDate ? ... : ... ;
return new RegExp(prefix + ...)
```

or just inline the whole thing.
```
return new RegExp(
  context.option.forwardDate ? `...` : `...` +
  ...
)
```

Reply:
Fixed 👍